internal/rules/resources_test.go

```diff
@@ -0,0 +1,64 @@
+package rules_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/titpetric/factory"
+
+	"github.com/crusttech/crust/internal/auth"
+	"github.com/crusttech/crust/internal/rules"
+	"github.com/crusttech/crust/system/types"
+)
+
+func TestRules(t *testing.T) {
```

Comment:
I do not see the point in testing if the DB works, should be more like service/repo struct we have elsewhere w/  proper unit testing.

Reply:
1. more tests > less tests
2. real environment > mocks

Most of the tests in internal/ (at least rbac and rules) are to verify functionality against real services, where mocking won't tell you if what actually is implemented works as it should. These aren't tests for the CI (but it would be obviously bad if they would break).

Mock tests will be added once the interface is finalized. I'm not 100% happy with the current interface as it can be simplified further (no need for Multi, there's a stutter with CheckAccess vs. just Check,...), but that's neither here nor there.

Reply:
I think this will be a learning moment for me: show me how ;)

Reply:
/slow/!short, sorry

```
	if testing.Short() {
		log.Println("skipping test in short mode.")
		return
	}
```

Reply:
Already done in main_test.go, I'll add the log info